src/photos/ducks/albums/components/AlbumsView.spec.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { mount } from 'enzyme'
+import PropTypes from 'prop-types'
+import AppLike from '../../../../../test/components/AppLike'
+import { mockedRouter } from '../../../../../test/__mocks__/mockedRouter'
+
+import AlbumsView from './AlbumsView'
+import { createMockClient } from 'cozy-client'
+
+describe('Albumsview', () => {
+  const client = createMockClient({})
+  let component
+  const setup = data => {
+    component = mount(
+      <AppLike client={client}>
+        <AlbumsView albums={{ data }} />
+      </AppLike>,
+      {
+        context: {
+          router: {
+            ...mockedRouter
+          }
+        },
+        childContextTypes: {
+          router: PropTypes.object
+        }
+      }
+    )
+  }
+  it('displays an empty Component', () => {
+    setup([])
```

Comment:
Same as in previous commit, I think naming the options would be more understandable, it's pretty hard to recall what a variable is meant in a call if its a boolean or an empty list.